eng/WpfArcadeSdk/tools/ApiCompat.targets

```diff
@@ -0,0 +1,55 @@
+<Project>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.ApiCompat" Version="$(MicrosoftDotNetApiCompatVersion)"/>
+  </ItemGroup>
+
+  <!-- Only enable API Compat when needed for some building assembly -->
+  <PropertyGroup Condition="'$(RunNetFrameworkApiCompat)'==''">
+    <RunNetFrameworkApiCompat>false</RunNetFrameworkApiCompat>
+    <RunNetFrameworkApiCompat Condition="'$(Net48CompatNeededProjects)'!='' 
+                                          and $(Net48CompatNeededProjects.Contains('$(MSBuildProjectName)'))">true</RunNetFrameworkApiCompat>
```

Comment:
This kind of pattern usually fails when dealing with generated projects, for e.g., `System.Windows.Controls.Ribbon_dklsjfl_wpftmp.csproj`

Reply:
Good call, I can add a scan for wpftmp as well.